ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Analyzers/LambdaAnalyzer.fs

```diff
@@ -84,8 +87,25 @@ type LambdaAnalyzer() =
             | x -> hasMatches, i = pats.Count, x
 
         compareArgsRec expr 0 null
+    
+    let isImplicitlyConvertedToDelegate (lambda: ILambdaExpr) =
+        let lambda = lambda.IgnoreParentParens()
+        let appTuple = TupleExprNavigator.GetByExpression(lambda)
+        let app = getArgsOwner lambda
```

Comment:
You go up through tuples twice, so it'll return a wrong arg owner in case of nested tuple argument.

Reply:
Sorry, I'm being wrong here.